apps/challenges/views.py

```diff
@@ -235,7 +235,9 @@ def challenge_detail(request, challenge_host_team_pk, challenge_pk):
         if serializer.is_valid():
             serializer.save()
             challenge = get_challenge_model(serializer.instance.pk)
-            serializer = ChallengeSerializer(challenge)
+            serializer = ChallengeSerializer(
```

Comment:
Let's not do these changes in this PR. Can you please open up a new PR with backend changes only